walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/pom.xml

```diff
@@ -0,0 +1,55 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
I think this file can be deleted, it may have been added to the PR when we were fighting with the "git rebase" issues.  I believe the pom.xml files are all created for you already, and you don't need to modify anything in the "walkthroughs" directory

Reply:
Addressed in 3.5part1